src/main/java/logisticspipes/LogisticsPipes.java

```diff
@@ -219,12 +220,17 @@ public LogisticsPipes() {
     private static LPGlobalCCAccess generalAccess;
     private static PlayerConfig playerConfig;
 
+    public static boolean isGTNH = false;
+
     @EventHandler
     public void init(FMLInitializationEvent event) {
 
         // Register Network channels
         MainProxy.createChannels();
 
+        // Gregtech New Horizons compat
+        isGTNH = Loader.isModLoaded("dreamcraft") && Loader.isModLoaded("gregtech");
```

Comment:
why check for GT too?

Reply:
It uses some gt specific classes and would crash otherwise, so felt safer.